README.md

```diff
@@ -1,82 +1,89 @@
 # react-validation-context
 
-Components for providing validation via React context.
-
-
-# Motivation
-
-There are several scenarios where a parent component's validation depends on whether or not its descendant elements validate. By
-using the [React `context` api][react-docs-context], manual crawling of the React render tree can be avoided. Instead, a handler
-function in the context can be called to update the parent's state whenever the descendant updates. This simplifies the
+There are several scenarios where a parent component's validation depends on
+whether or not its descendant elements validate. By using the [React `context`
+api][react-docs-context], manual crawling of the React render tree can be
+avoided. Instead, a handler function in the context can be called to update the
+parent's state whenever the descendant updates. This simplifies the
 implementation of validation in the parent component.
 
-
-# Install
+## Install
 
 This package is available on `npm`. Install it using:
 
 ```sh
-npm install react-validation-context --save
+npm install --save react-validation-context
```

Comment:
Could drop `--save` altogether since it's the default now.

Reply:
DID YOU JUST ASSUME MY NPM VERSION??